Utilities/Test-VSProjectConsistency/Test-VSProjectConsistency.psm1

```diff
@@ -104,47 +104,47 @@ function Test-VSProjectConsistency
         # Return with a message if there aren't any .csproj files to process.
         if ($projectFiles.Length -eq 0)
         {
-            Write-Output "No .csproj in the folder."
+            Write-Output 'No .csproj in the folder.'
             return
         }
 
         # The default whitelist of the extensions to we're interested in.
         $fileExtensions = @(
-            ".config",
-            ".cs",
-            ".cshtml",
-            ".css.map",
-            ".css",
-            ".gif",
-            ".ico",
-            ".info",
-            ".jpeg",
-            ".jpg",
-            ".js.map",
-            ".js",
-            ".less",
-            ".min.css",
-            ".min.js",
-            ".png",
-            ".ts")
-        $fileExtensionsForRegex = "(" + ($fileExtensions -join "|") + ")$"
+            '.config',
+            '.cs',
+            '.cshtml',
+            '.css.map',
+            '.css',
+            '.gif',
+            '.ico',
+            '.info',
+            '.jpeg',
+            '.jpg',
+            '.js.map',
+            '.js',
+            '.less',
+            '.min.css',
+            '.min.js',
+            '.png',
+            '.ts')
+        $fileExtensionsForRegex = '(' + ($fileExtensions -join '|') + ')$'
```

Comment:
Could be made into a single expression.
```suggestion
        $fileExtensionsForRegex = "($($fileExtensions -join '|'))`$"
```

Could be simplified:
![image](https://user-images.githubusercontent.com/4007293/213821804-35e075dd-7666-4a1d-9c6d-5a7cdcb8793e.png)